### ISSUE TITLE
First commits has wrong dates

### ISSUE THREAD

**Issue by bheisig** [c3e/grundgesetz#1]
The Grundgesetz was first announced on 1949-05-23. But as you may notice the first commit dates are not older than 1970-01-01. Commit dates are based on the [Unix time](https://en.wikipedia.org/wiki/Unix_time) which starts at that date. The Unix time is defined as a number of seconds. Git (and GitHub, too) has a limitation that only positive numbers of seconds is parsed correctly. For a Unix time before 1970-01-01 we need negative numbers of seconds to be allowed.

**tl;dr:** We are currently unable to fix some commit dates due to technical limitations.

**Comment by floscher** [comment]
By now it is only a GitHub problem, since git now supports pre-1970 dates as well as dates in the far future (see https://github.com/Legilibre/Archeo-Lex/issues/47). The web viewer of the French legislature project shows commits from 1803: [git blame view of the French "Code Civil"](https://archeo-lex.fr/codes/code_civil/2023-05-21/annotations)

For pushing to a git remote, the settings of that remote repository need to be adapted (see https://github.com/c3e/grundgesetz-dev/issues/28).
